src/error.rs

```diff
@@ -143,6 +144,29 @@ impl HandlerErrorKind {
     pub fn is_sentry_event(&self) -> bool {
         !matches!(self, HandlerErrorKind::InvalidUA)
     }
+
+    pub fn as_response_string(&self) -> String {
+        match self {
+            HandlerErrorKind::General(_) | HandlerErrorKind::Internal(_) => self.to_string(),
```

Comment:
There were a couple places I thought might be using `Internal` while potentially exposing strings we don't want exposed, but it looks like they're all either called at startup (e.g. StoreImage::check_bucket`) or also the `AdmFilter` periodic updater that runs on its own thread.

Makes me wonder if we should move to using something else other than `Internal` for startupy errors in the future. Maybe `check_bucket` failing is really more of a `config::ConfigError` like we've started using elsewhere?

Reply:
`CloudStorage` could happen later, if the storage ever breaks for some random reason, but yeah, most of the others are either a `ConfigError` or something start-up related at least. Those probably don't need dedicated messages. 

We can add a ticket to review the errors we're recording at some later date.